src/rulesets/oas/functions/oasExample.ts

```diff
@@ -0,0 +1,78 @@
+import type { IFunction, IFunctionContext, IFunctionResult } from '../../../types';
+import type { Optional, Dictionary } from '@stoplight/types';
+import type { ISchemaOptions } from '../../../functions/schema';
+
+function isObject(value: unknown): value is Dictionary<any> {
+  // todo: expose a util, so it can be used everywhere
+  return value !== null && typeof value === 'object';
```

Comment:
@P0lip or maybe just use lodash `isOjbectLike`?

Reply:
Yeah, I considered doing it, but hm, tree-shaking feels shaky :laughing: 
Let me see how it impacts the size of the bundled fn.
It's the difference is negligible, I'll go for isObjectLike